### PR TITLE
chore: housekeeping on things that we dont want

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -47,7 +47,6 @@ module.exports = {
     }]
   },
   externals: {
-    codemirror: 'CodeMirror',
     'loop-protect': 'loopProtect'
```
Comment (Bouncey) [inline webpack.config.js]: We are no using `loop-protect` any more

**Review by Bouncey — CHANGES_REQUESTED** [review]
Will this also get rid of the optimizely cookies?

File: webpack.config.js (hunk 1 above)
Comment (raisedadead) [reply webpack.config.js]: I know. The PR is WIP

**Comment by raisedadead** [comment]
> Will this also get rid of the optimizely cookies?

Not the ones already set in the browser. New will not be added.

**Comment by raisedadead** [comment]
#### Please rebase and merge in case we want to revert a specific commit

**Review by Bouncey — CHANGES_REQUESTED** [review]
We could also remove the build step for `frame-runner` in the gulp file and the `webpack.frame-runner.js`

**Comment by QuincyLarson** [comment]
@raisedadead This all looks good. It needs to be tested. @Bouncey when you get a moment could you deploy this to freecodecamp.rocks and we can verify over time that it doesn't seem to have broken anything?

**Review by sittirac1234 — APPROVED** [review]

**Review by Bouncey — APPROVED** [review]

**Pull request merged**